_topic_maps/_topic_map.yml

```diff
@@ -1600,6 +1600,8 @@ Topics:
     File: authenticating-pipelines-using-git-secret
   - Name: Viewing pipeline logs using the OpenShift Logging Operator
     File: viewing-pipeline-logs-using-the-openshift-logging-operator
+  - Name: Running workload as user namespaces on OpenShift Pipelines
```

Comment:
The title says workloads but the rest of doc is all about the "buildah" task. We should explicitly talk about both workloads and buildah in particular.